metadata-ingestion/setup.py

```diff
@@ -349,6 +349,9 @@
         # https://github.com/great-expectations/great_expectations/pull/6149.
         "great-expectations != 0.15.23, != 0.15.24, != 0.15.25, != 0.15.26",
     },
+    "hive-metastore": sql_common
```

Comment:
let's add a comment on the `presto-on-hive` one that it should be kept in sync with this